packages/framework-provider-azure/package.json

```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "@azure/cosmos": "^4.0.0",
     "@azure/functions": "^1.2.2",
-    "@azure/identity": "~2.1.0",
+    "@azure/identity": "~4.7.0",
```

Comment:
Is there any API change in this version upgrade that requires taking something into account on this provider?